### PR TITLE
Clean up on the phrasing and grammar

### PR DESCRIPTION
**Pull request by stramel**
Just a handful of things that I think will improve the English version of the docs

### CODE REVIEW

File: src/components/codeExamples/defaultExample.ts [hunk 1]
```diff
@@ -8,9 +8,9 @@ function YourForm() {
   console.log(watch('example')) // watch input value by passing the name of it
 
   return (
-    {/* handleSubmit will validation your inputs before invoking onSubmit */}
+    {/* `handleSubmit` will validate your inputs before invoking `onSubmit` */}
```
Comment (stramel) [inline src/components/codeExamples/defaultExample.ts]:
```suggestion
{/* "handleSubmit" will validate your inputs before invoking "onSubmit" */}
```

File: src/components/codeExamples/defaultExample.ts [hunk 2]
```diff
@@ -8,9 +8,9 @@ function YourForm() {
...
     <form onSubmit={handleSubmit(onSubmit)}>
-      {/* register your input into Hook by invoking the register function */}
+      {/* register your input into the hook by invoking the `register` function */}
```
Comment (stramel) [inline src/components/codeExamples/defaultExample.ts]:
```suggestion
{/* register your input into the hook by invoking the "register" function */}
```

File: src/components/ApiPage.tsx [hunk 3]
```diff
@@ -398,9 +398,9 @@ function Builder({ formData, showApi }: any) {
   }
 })
 
-<input name="firstName" ref={register} /> // ✅ works version
+<input name="firstName" ref={register} /> // ✅ working version
 <input name="lastName" ref={() => register({ name: 'lastName' })} /> 
-// ❌ above example does not work with defaultValues due to its ref is not provided
+// ❌ above example does not work with defaultValues due to its `ref` not being provided
```
Comment (stramel) [inline src/components/ApiPage.tsx]:
```suggestion
// ❌ above example does not work with "defaultValues" due to its "ref" not being provided
```

**Comment by bluebill1049** [comment]
TOO GOOD! thanks so much, @stramel

**Comment by stramel** [comment]
lol Thanks @bluebill1049!

**Pull request merged**